sdk/core/generic-core/generic/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
```

Comment:
Is there a need to nest this inside a namespace?
Are we thinking this will be a new group of packages?
I would somewhat prefer this be a flat package like msrest was unless there's a need for a new group namespace.

Reply:
Flattened for now, but naming still TBD!